ext/MitsubaLoader/CSerializedLoader.cpp

```diff
@@ -206,17 +221,61 @@ asset::SAssetBundle CSerializedLoader::loadAsset(io::IReadFile* _file, const ass
 		auto meshBuffer = core::make_smart_refctd_ptr<asset::ICPUMeshBuffer>();
 		meshBuffer->setPositionAttributeIx(POSITION_ATTRIBUTE);
 
-		/*
-			TODO after merging CPLY Pull Request (required default Assets):
+		auto makeAvailableAttributesVector = [&]()
+		{
+			core::vector<uint8_t> vec;
```

Comment:
reserve 4 before pushing anything

Reply:
why? I doesn't make any sense in that case and that purpose I think

Reply:
since u know max number of pushes, with reserve u're making sure it'llbe allocating only once. Common optimization

Reply:
yeah I have read std::vector reserve and it's fine now